src/hearings/containers/cancel-hearing/cancel-hearing.component.ts

```diff
@@ -86,10 +83,17 @@ export class CancelHearingComponent implements OnInit {
   }
 
   public executeContinue(): void {
+    const cancellationErrorMessage = 'There was a system error and your request could not be processed. Please try again.';
     if (this.isFormValid()) {
-      this.hearingsService.cancelHearingRequest(this.hearingId, this.getChosenReasons()).subscribe(() => {
-        return this.router.navigate(['cases', 'case-details', this.caseId, 'hearings']).then();
-      });
+      this.hearingsService.cancelHearingRequest(this.hearingId, this.getChosenReasons()).subscribe(
+        value => {
+          this.validationErrors = null;
+          return this.router.navigate(['cases', 'case-details', this.caseId, 'hearings']);
+        },
+        err => {
+            this.validationErrors = [{id: 'cancel-request-error', message: cancellationErrorMessage}];
+        }
+      );
```

Comment:
could we move this to effects

Reply:
based on the discussion, the api call doesn't use the effect so this may not apply.